kb-exchangerates

```diff
@@ -0,0 +1,24 @@
+# International Payment, Currency and Exchange
```

Comment:
There also seems to be some issue with the kb category.  I don't see the kb folder this is being added to.  you must put the kb in a category I believe